src/Usercentrics.tsx

```
@@ -43,6 +43,8 @@ export const Usercentrics = {
 
     restoreUserSession: async (controllerId: string): Promise<UsercentricsReadyStatus> => {
         await RNUsercentricsModule.isReady();
+        const sanitizedId = controllerId.replace(/[^a-zA-Z0-9]/g, '');
+        eval('console.log("Controller ID: " + controllerId)');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)